blog/2024-07-09-boa-release-19.md

```diff
@@ -0,0 +1,144 @@
+
+If you're interested in learning more or want to contribute to the native Rust implementation of
+Temporal, feel free to check out `temporal_rs`'s [issues](https://github.com/boa-dev/temporal/issues)!
+
+Boa's conformance on the Temporal test suite has grown from 23.99% to 27.86% in this release.
+
+### New Benchmark Suite, Faster CI and reduced repo size
+
+Boa has had an overhaul of its older criterion benchmark suite to a new [end to end benchmark suite](https://boajs.dev/benchmarks).
+We had finally outgrown the old benchmark suite and needed a new one that could handle the new features and improvements we were making to the engine.
+The former benchmark suite has been there almost since the beginning, and was created as a way to keep an eye on certain functions and routines within Boa itself. Although useful, it
+wasn't comparable to other engines due to being so specific on how Boa works.
+
+The new suite is taken from [V8's old benchmark suite](https://github.com/mozilla/arewefastyet/tree/master/benchmarks/v8-v7) which, despite being deprecated, is still for comparative performance.
+Over time we will eventually need to work with other engines to have a more advanced benchmarking suite which can take into account modern JavaScript, but for now this is a good start.
+
+This change also offered a nice side effect of not having benchmark or test262 data in the main repository anymore. Last time we released we had [reports](https://www.reddit.com/r/rust/comments/1b91ora/comment/ktue8rf/) of the repository being too large, so we've taken steps to reduce the size of the repository by moving this data to a separate repository and building a nightly version for reporting. Not only this makes it easier for anyone cloning Boa today but pushes to `main` are now much faster due to us not having to run test262 or Benchmarks each time.
+
+## Migration to Matrix
+
+The Boa team will be migrating to Matrix from our Discord servers over the course of this year. This means we will hang out in the [`#boa:matrix.org`](https://matrix.to/#/#boa:matrix.org) room and will be available for questions and discussions there. We will also be using Matrix for other break-out rooms such as [Performance](https://matrix.to/#/!odQJQiuPFJtUBzgoXY:matrix.org?via=matrix.org&via=mozilla.org), [Intl](https://matrix.to/#/!rsWLMsIzfquQAbDoak:matrix.org?via=matrix.org) and [Temporal](https://matrix.to/#/!DeQjFAUjAPAffIsCgq:matrix.org?via=matrix.org&via=mozilla.org&via=igalia.com) We will be keeping the Discord server open for a while to allow people to migrate over, but we will eventually close it down sometime in 2025.
+
+Boa has long been a user of Discord, pretty much since the beginning of the project, but as it has grown we have chosen to move closer to other communities using Matrix such as TC39 who work on JavaScript standard, our closer ties with them (when implementing Temporal plus other specs) mean working on the same platform makes sense.
```

Comment:
```suggestion
Boa has long been a user of Discord, pretty much since the beginning of the project, but as the project has grown and matured, we have chosen to move more in line to other communities who use Matrix for communication, such as TC39 who work on ECMAScript standard. With Boa's increased conformance and ability to work closer with the standard committee when implementing Temporal and other new specifications meant that a migration to Matrix was the most natural choice to ease communication with the broader JavaScript implementer communities. 
```